dask_gdf/accessor.py

```diff
@@ -96,3 +97,20 @@ def _validate(self, series):
         if not isinstance(series._meta._column, gd.datetime.DatetimeColumn):
             raise AttributeError("Can only use .dt accessor with datetimelike "
                                  "values")
+
+
+class CategoricalAccessor(Accessor):
+    """ Accessor object for categorical properties of the Series values
+    of Categorical type.
+    """
+
+    from pygdf.categorical import CategoricalAccessor as gdfCategoricalAccessor
```

Comment:
Please use PEP8 naming conventions.  `gdfCategoricalAccessor` should be `GdfCategoricalAccessor`